src/components/modals/ListModal.tsx

```diff
@@ -70,7 +72,19 @@ export function ListModal<T>({
   closeArrow = true,
   onSubmitEditing,
   onViewableItemsChanged,
-  ...textProps
+  // SimpleTextInput props:
+  iconComponent = SearchIconAnimated,
+  placeholder,
+  autoCorrect = false,
+  autoCapitalize = 'words',
+  returnKeyType = 'search',
+  keyboardType,
+  blurOnSubmit,
+  inputAccessoryViewID,
+  maxLength,
+  secureTextEntry,
+  autoFocus,
+  blurOnClear = true
```

Comment:
Blur on clear is actually a very bad UX. I know it's not a part of this task, but please default to `false`